cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Student Visibility') %></h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="staff_only">
+                <%- gettext('Hide from students') %>
```

Comment:
Looking closer at your `Hide content from learners` suggestion - there's actually a large difference between this option and the one that follows.

- `Hide from learners` will hide the subsection's content *and* everything else about the subsection (from the student POV, it does not exist at all).
- `Hide content but not grades` (or however it ends up being labelled) will still allow students to see the subsection (it exists), but the contents are hidden via @nasthagiri's work on the LMS PR.

So including `content` was intended to make that differentiation clear.

What about options like:
- Hide Entire Subsection
- Hide Contents After Due Date
- Nothing Hidden

and then further details are given in the tip?